sql/2020/05_Media/video_source_types.sql

```diff
@@ -3,13 +3,13 @@
 
 SELECT
   client,
-  video_type,
+  lower(video_type) as video_type,
```

Comment:
CAPS for all keywords and functions
```suggestion
  LOWER(video_type) AS video_type,
```